Remotion/Development/Sandboxing.NUnit2/Development.Sandboxing.NUnit2.csproj

```diff
@@ -17,7 +17,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Remove="NUnit" />
-    <PackageReference Include="NUnit" Version="2.6.3" />
+    <PackageReference Include="NUnit" Version="4.1.0" />
```

Comment:
Upgrading NUnit2 to NUnit4 is not intended. The project is explicitly named "NUnit2"